client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml

```diff
@@ -0,0 +1,44 @@
+baseClass: "com.paypal.selion.testcomponents.BasicPageImpl"
+defaultLocale: "US"
+elements:
+  fieldXTextField:
+    locators:
+      US: "//input[@id='fieldXId']"
+      FR: "//input[@id='fieldXId_FR']"
+  continueButton:
+    locators:
+      US: "//input[@id='no_such_button']"
+  hiddenButton:
+    locators:
+      US: "//input[@id='hidden.x']"
+  logLabel:
+    locators:
+      US: "//label[@id='logLabel']"
+  xCheckBox:
+    locators:
+      US: "//input[@id='checkbox']"
+  xRadioButton:
+    locators:
+      US: "//input[@id='radio']"
+  xSelectList:
+    locators:
+      US: "//select[@id='select']"
+  selionContainer:
+    locators:
+      US: "//div"
+    containerElements:
+      containerButton:
+        locators:
+          US: ".//input"
+pageTitle:
+  US: "TestPage Generated By JavaScript"
+pageValidators: [
+  hiddenButton.isNotVisible,
+  "!hiddenButton.isVisible",
+  "!continueButton",
+  hiddenButton.is!Visible,
```

Comment:
Don't think we need to support the `!` in the middle of these two words. :) 

It just looks odd.